### PR TITLE
x11-terms/roxterm: Bump to 3.9.2

### PR DESCRIPTION
**Pull request by ekohl**
Closes: https://bugs.gentoo.org/581272
Closes: https://bugs.gentoo.org/718548
Package-Manager: Portage-2.3.89, Repoman-2.3.20

### CODE REVIEW

**Comment by ekohl** [comment]
While testing this, I did run into https://github.com/realh/roxterm/issues/222. The patch is submitted as https://github.com/realh/roxterm/pull/223.

**Comment by ekohl** [comment]
As indicated in https://bugs.gentoo.org/581272#c3, I am interested in becoming a proxy maintainer. First I want to dig in the odd character issue because it's pretty annoying. In https://github.com/realh/roxterm/issues/222#issuecomment-617449057 I have traced it to the script provided by `x11-libs/vte-0.58.3` and come up with a workaround but I'm going to see if a newer vte would solve it.

**Comment by mgorny** [comment]
Then please don't be shy and add yourself to metadata.xml. You can also add a commit removing p.mask entry.

**Comment by ekohl** [comment]
Will do. Is it ok to keep this open while I track down the issue with vte and then update this? In the mean time I'm testing this new version via my personal overlay.

**Comment by ekohl** [comment]
Something like this? I added one commit to add myself as proxy maintainer which also removes The package.mask. The other was updated to depend on vanilla vte as well as update the metadata remote-id. I wasn't sure how to split up commits and conventions in the repo.

File: profiles/package.mask [hunk 1]
```diff
@@ -127,11 +127,6 @@ media-gfx/svg2rlg
 # release. Bug 718326. Removal whenever it becomes an issue.
 mail-filter/pyzor
 
-# Michał Górny <mgorny@gentoo.org> (2020-04-19)
```
Comment (juippis) [inline profiles/package.mask:130]: Please split this to a 3rd commit.

File: x11-terms/roxterm/metadata.xml [hunk 2]
```diff
@@ -11,6 +11,6 @@
 	</maintainer>
 
 	<upstream>
-		<remote-id type="sourceforge">roxterm</remote-id>
+		<remote-id type="github">realh/roxterm</remote-id>
```
Comment (juippis) [inline x11-terms/roxterm/metadata.xml:14]: You can do this edit in while changing metadata.xml.

File: x11-terms/roxterm/roxterm-3.9.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+DESCRIPTION="A terminal emulator designed to integrate with the ROX environment"
+HOMEPAGE="https://github.com/realh/roxterm"
+SRC_URI="https://github.com/realh/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-3"
+SLOT="1"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-libs/dbus-glib-0.100
```
Comment (juippis) [inline x11-terms/roxterm/roxterm-3.9.2.ebuild:16]: Version restriction can be lifted.

File: x11-terms/roxterm/roxterm-3.9.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	dev-libs/glib:2
+	dev-libs/libpcre2
+	sys-apps/dbus
+	x11-libs/gtk+:3
+	x11-libs/vte:2.91[vanilla]
+	!x11-terms/roxterm:0"
```
Comment (juippis) [inline x11-terms/roxterm/roxterm-3.9.2.ebuild:22]: This blocker can be removed, as I can see :0 hasn't existed in years.

File: x11-terms/roxterm/roxterm-3.9.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+# vanilla vte due to https://github.com/realh/roxterm/issues/222
+# slotted versions of roxterm need block one another due to file collisions. #565288
+DEPEND="${RDEPEND}
+	dev-libs/libxslt
+	virtual/pkgconfig"
```
Comment (juippis) [inline x11-terms/roxterm/roxterm-3.9.2.ebuild:27]: This is BDEPEND.

File: x11-terms/roxterm/roxterm-3.9.2.ebuild (hunk 3 above)
Comment (ekohl) [reply x11-terms/roxterm/roxterm-3.9.2.ebuild:16]: What are the reasons for this? Isn't it good practice to at least state the oldest version if there is one?

**Comment by ekohl** [comment]
Updated to address comments. Also updated it to the now released 3.9.3 and dropped PATCHES since https://github.com/realh/roxterm/pull/223 is included.

File: x11-terms/roxterm/roxterm-3.9.2.ebuild (hunk 3 above)
Comment (juippis) [reply x11-terms/roxterm/roxterm-3.9.2.ebuild]: Then you should specify minimum requirements for every dep, not just one. It's just a lot cleaner and easier to maintain this way.

**Comment by juippis** [comment]
You need to create a bugzilla account with that e-mail you just put to metadata.xml so all bugs get assigned to you. If you already have a bugzilla account, just switch it to that e-mail.

**Comment by ekohl** [comment]
Updated my Bugzilla email address.

**Pull request closed without merging**